app/templates/components/github-apps-repository.hbs

```diff
@@ -4,9 +4,43 @@
   {{name}}
 {{/link-to}}
 
-{{#unless repository.active_on_org}}
+{{#if repository.active_on_org}}
+  {{#if repository.permissions.migrate}}
+    {{#if repository.migrationStatus}}
+      <div
+        data-test-migration-status={{repository.migrationStatus}}
+        class="migration-status {{repository.migrationStatus}}"
+      >
+      {{#if (eq repository.migrationStatus "migrating")}}
```

Comment:
I would avoid using helpers wherever possible due to performance issues (helpers calculate values eagerly and don't cache results).

Reply:
I sometimes avoid helpers for clarity, but I don't think that we need to think about performance in such cases. Checking equality will not add any considerable overhead.

Reply:
Regarding clarity, what I meant is that `#if isMigrating` or `#if migrationSucceeded` looks better than `eq`.

Reply:
@drogus I agree about clarity, but disagree about performance. Even in this case, this template is a part of component, which is used in a loop over user's GitHub Apps repositories. Thus, the value would be recalculated on each re-render of each repository. It's ok if a user has 10 repositories, but in cases with hundreds of repos on low-end devices this overhead may be pretty noticable. Also it's hard to predict the frequency of re-renders.

Reply:
I'd say that without benchmarking you don't really know if you're making things worse or better, but in the same time I can't prove anything either.

Reply:
Relevant: https://simplabs.com/blog/2017/03/21/on-computed-properties-vs-helpers.html. 
From that article, it seems like you mostly need to worry about eager evaluation *of the arguments*, so it doesn't seem like this would be a bad use case for a helper. I'll leave it as-is for now and we can address the topic more generally in a subsequent PR if we decide to avoid these types of template helpers altogether.